src/components/Info/Info.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+function createArrayFromPhrase(phrase) {
+  const splitPhrase = phrase.split(' ');
+  const thirdWord = splitPhrase.pop();
+  return [splitPhrase.join(' '), thirdWord];
+}
+
+const InfoSection = props => (
+  <section className={`bx--row ${props.className} info-section`}>
+    <div className="bx--col-md-8 bx--col-lg-4 bx--col-xlg-3">
+      <h3 className="info-section__heading">{props.heading}</h3>
+    </div>
+    {props.children}
+  </section>
+);
+
+const InfoCard = props => {
+  const splitHeading = createArrayFromPhrase(props.heading);
+
+  return (
+    <div className="info-card bx--col-md-4 bx--col-lg-4 bx--col-xlg-3 bx--offset-xlg-1">
+      <h4 className="info-card__heading">
+        {`${splitHeading[0]}`}
```

Comment:
You're missing the space between the second and third words. See https://www.carbondesignsystem.com/tutorial/react/step-4/#type